### ISSUE TITLE
enable media playback within a conversation

### ISSUE THREAD

**Issue by sudwhiwdh** [nextcloud/spreed#5280]
Use case for me would be to be able to play an audio or video file or embedded platform video in Nextcloud Talk in a conversation so that all participants can listen/view it in parallel.
This will be a welcome function, especially in the now mostly digital education work in schools and universities.

Related Issues for Android & iOS
nextcloud/talk-android#1043
https://github.com/nextcloud/talk-ios/issues/510

**Comment by nickvergessen** [comment]
Duplicate of #4658
